backends/occa/ceed-occa-basis.c

```diff
@@ -108,6 +112,22 @@ static int CeedBasisDestroy_Occa(CeedBasis basis) {
   return 0;
 }
 
+// *****************************************************************************
+/// OCCA Backend: Create a tensor product basis for H^1 discretizations
+///
+/// @param ceed       Ceed
+/// @param dim        Topological dimension
+/// @param P1d        Number of nodes in one dimension
+/// @param Q1d        Number of quadrature points in one dimension
+/// @param interp1d   Row-major Q1d × P1d matrix expressing the values of nodal
+///                   basis functions at quadrature points
+/// @param grad1d     Row-major Q1d × P1d matrix expressing derivatives of nodal
+///                   basis functions at quadrature points
+/// @param qref1d     Array of length Q1d holding the locations of quadrature
+///                   points on the 1D reference element [-1, 1]
+/// @param qweight1d  Array of length Q1d holding the quadrature weights on the
+///                   reference element
+/// @param[out] basis New basis
```

Comment:
So this comment block will appear in Doxygen. Either the this should be demoted to `//` like the one above or that one should be promoted to `///` and we figure out how to ensure that (a) docs are always kept consistent between the interface and implementation (backend) and (b) implementation docs do not clutter what the user sees. This block looks like a copy/paste, which I don't like because it makes change harder and inevitably becomes inconsistent.